### PR TITLE
Add new volunteer Prakhar Tiwari

### PR DESCRIPTION
**Pull request by prakhartiwari0**

### CODE REVIEW

File: data/users/prakhartiwari0.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Prakhar Tiwari",
+  "bio": "Student Web Developer | Tech Creator",
+  "image": "https://avatars.githubusercontent.com/u/65062036?v=4",
+  "categories": [
+    {
+      "slug": "programming | web-development",
```
Comment (RakeshSangem) [inline data/users/prakhartiwari0.json:7]: Please only one slug. We don't the feature to use multiple slugs yet.

Comment (prakhartiwari0) [reply data/users/prakhartiwari0.json:7]: Okay sure

**Comment by prakhartiwari0** [comment]
Oh no start review by mistake

File: data/users/prakhartiwari0.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Prakhar Tiwari",
+  "bio": "Student Web Developer | Tech Creator",
+  "image": "https://avatars.githubusercontent.com/u/65062036?v=4",
+  "categories": [
+    {
+      "slug": "web-development",
+      "topics": ["javascript","html", "css", "python", "git", "github", "open-source"]
```
Comment (codewithvoid) [inline data/users/prakhartiwari0.json:8]: keep it to max 3 technical topics for now.

File: data/users/prakhartiwari0.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Prakhar Tiwari",
+  "bio": "Student Web Developer | Tech Creator",
```
Comment (codewithvoid) [inline data/users/prakhartiwari0.json:3]: Would be better if you can pick one. Also just `Web Developer` is fine

File: data/users/prakhartiwari0.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (codewithvoid) [inline data/users/prakhartiwari0.json:1]: we are yet to add lint check

can you ensure json is formatted using https://jsonformatter.org

**Comment by prakhartiwari0** [comment]
I have done the required changes, and have checked the JSON syntax in https://jsonformatter.org and it is showing "Valid JSON". Kindly check now

**Review by RakeshSangem — APPROVED** [review]

**Comment by codewithvoid** [comment]
Thanks for making the changes Prakhar!

**Review by codewithvoid — APPROVED** [review]

**Pull request merged**